lib/src/ui/common/util/mappers.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:allthenews/src/ui/pages/home/news/primary_news/primary_news_list_entity.dart';
+import 'package:allthenews/src/ui/pages/home/news/secondary_news/secondary_news_list_entity.dart';
+
+extension PrimaryNewsEntityMapper on List<PrimaryNewsListEntity> {
```

Comment:
imo, jak to jest używane w jednym miejscu to chyba dałbym jako prywatny extension w pliku, w którym został użyty :)